src/utils/effortColors.js

```diff
@@ -24,4 +49,4 @@ export const getprogress = (effort) => {
   return progress;
 };
 
-export default getcolor;
+export default {getProgressColor,getcolor};
```

Comment:
1. Can you please include comment on why/where it is being used (eg. Leaderboard, tasks, etc) ?  Because its confusing to see 2 different functions names with similar functionality. 
2. Is the export statement at the end partially updated or is it intentional ? I am thinking if "{getProgressColor,getcolor}" should be "{ getProgressColor, getProgressValue }" ?